### PR TITLE
Issue #9391: update example of AST for TokenTypes.ANNOTATION_DEF 

### LINKED ISSUE #9391: update example of AST for TokenTypes.ANNOTATION_DEF
https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ANNOTATION_DEF

needs to be extended to show example of AST in format that printed by our CLI (change "+-" to "`-").
Source to be updated: https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

PLEASE READ for more details: https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-AST-in-TokenTypes

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
fixes #9391 
picture

Java Code used here:
<pre>
<code>
public class MyClass {
  public @interface MyAnnotation {
    int somevalue();
  }
}
 
</code>
</pre>

AST Format of the Java Doc:
<pre>
<code>
CLASS_DEF -> CLASS_DEF [1:0]
|--MODIFIERS -> MODIFIERS [1:0]
|   `--LITERAL_PUBLIC -> public [1:0]
|--LITERAL_CLASS -> class [1:7]
|--IDENT -> MyClass [1:13]
`--OBJBLOCK -> OBJBLOCK [1:21]
    |--LCURLY -> { [1:21]
    |--ANNOTATION_DEF -> ANNOTATION_DEF [2:2]
    |   |--MODIFIERS -> MODIFIERS [2:2]
    |   |   `--LITERAL_PUBLIC -> public [2:2]
    |   |--AT -> @ [2:9]
    |   |--LITERAL_INTERFACE -> interface [2:10]
    |   |--IDENT -> MyAnnotation [2:20]
    |   `--OBJBLOCK -> OBJBLOCK [2:33]
    |       |--LCURLY -> { [2:33]
    |       |--ANNOTATION_FIELD_DEF -> ANNOTATION_FIELD_DEF [3:4]
    |       |   |--MODIFIERS -> MODIFIERS [3:4]
    |       |   |--TYPE -> TYPE [3:4]
    |       |   |   `--LITERAL_INT -> int [3:4]
    |       |   |--IDENT -> somevalue [3:8]
    |       |   |--LPAREN -> ( [3:17]
    |       |   |--RPAREN -> ) [3:18]
    |       |   `--SEMI -> ; [3:19]
    |       `--RCURLY -> } [4:2]
    `--RCURLY -> } [5:0]
</code>
</pre>

Expected Java Doc to changes:
<pre>
<code>

|   |--MODIFIERS -> MODIFIERS 
|   |--TYPE -&gt; TYPE 
|   |   `--LITERAL_INT -&gt; int 
|   |--IDENT -&gt; somevalue 
|   |--LPAREN -&gt; ( 
|   |--RPAREN -&gt; ) 
|   `--SEMI -&gt; ; 
`--RCURLY -&gt;
</code>
</pre>

### CODE REVIEW

**Comment by Rahulkhinchi03** [comment]
@romani can you please help me how can I resolve this failing issue?

**Comment by Rahulkhinchi03** [comment]
@pbludov or @romani  please review my changes and approve for the same!

**Review by AkMo3 — APPROVED** [review]

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -4399,28 +4399,26 @@
...
+     *  |--AT -&gt; @
+     *  |--LITERAL_INTERFACE -&gt; interface
+     *  |--IDENT -&gt; MyAnnotation
+     *  `--OBJBLOCK -&gt; OBJBLOCK
+     *      |--LCURLY -&gt; {
+     *      |--ANNOTATION_FIELD_DEF -&gt; ANNOTATION_FIELD_DEF
+     *      |   |--MODIFIERS -&gt; MODIFIERS
+     *      |   |--TYPE -&gt; TYPE
+     *      |   |   `--LITERAL_INT -&gt; int
+     *      |   |--IDENT -&gt; value
```
Comment (timurt) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:4414]: someValue

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 2]
```diff
@@ -4399,28 +4399,26 @@
...
+     *      |   |--LPAREN -&gt; (
+     *      |   |--RPAREN -&gt; )
+     *      |   |--LITERAL_DEFAULT -&gt; default
+     *      |   |   `--EXPR -&gt; EXPR
+     *      |   |       `--NUM_INT -&gt; 1
```
Comment (timurt) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:4419]: there is `default` in example
```
     * public @interface MyAnnotation     
     * {     
     *     int someValue();    
     * }
```

please use old example and update AST

**Review by timurt — CHANGES_REQUESTED** [review]

**Pull request closed without merging**